azerothcore/azerothcore-wotlk
# fix(DB/ElwynnForest): allow Defias Cutpurse to use their Backstab ability

## Changes Proposed:
- add 2 castflag to allow them to use their backstab ability which usually costs 60 energy, which they don't have

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/16433

## SOURCE:


Ability is in db. Mob cannot cast it because of energy requirements 

## Tests Performed:

- tested ingame. works
- 


## How to Test the Changes:


1. Go to Elwynn Forest towards the north where the Defias Cutpurse camps are
2. Turn your back
3. Should get backstabbed

## Known Issues and TODO List:


It could technically be possible they use a different backstab version. But that would require sniffing. According to wowhead they do seem to use a backstab that uses energy.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
